log/util_test.go

```diff
@@ -0,0 +1,86 @@
+package log
+
+import (
+	"errors"
+	"io"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var nilInterface io.Reader
```

Comment:
this can be scoped inside the test function

Reply:
Sure ! Will do